.github/workflows/e2e-stress-test-flake-fix.yml

```diff
@@ -47,9 +47,6 @@ jobs:
       QA_DB_ENABLED: true
       CYPRESS_QA_DB_MONGO: true
       CYPRESS_REPLAYIO_ENABLED: 1
-      CYPRESS_REPLAYIO_ENABLE_UPLOAD: 0
```

Comment:
This doesn't work.
`false` doesn't work either.

The only solution was to omit it.